.gitignore

```diff
@@ -19,6 +19,7 @@ __pycache__/
 # C extensions
 *.so
 
+
```

Comment:
If the only change in this file is an added blank line, let's remove the blank line.

Reply:
I have changed nothing in this file.
Kindly remove the blank if possible .

Reply:
You can remove the blank line in your source and then commit and push the changes.

Reply:
Sir can you do me a favour?

Reply:
I am having some issues with my git!
I want help regarding that.